library/common/types/c_types.h

```diff
@@ -153,6 +153,8 @@ typedef struct {
   int64_t connection_id;
   // The number of internal attempts to carry out a request/operation. 0 if not present.
   uint64_t attempt_count;
+  // The number of bytes sent upstream.
+  uint64_t sent_byte_count;
```

Comment:
I don't think adding this has any effect unless you add it to 
https://github.com/envoyproxy/envoy-mobile/blob/main/library/common/jni/jni_utility.cc#L93
https://github.com/envoyproxy/envoy-mobile/blob/main/library/swift/StreamIntel.swift#L25
etc.